cedar-policy-core/Cargo.toml

```diff
@@ -28,6 +28,7 @@ stacker = "0.1.15"
 arbitrary = { version = "1", features = ["derive"], optional = true }
 miette = { version = "7.1.0", features = ["serde"] }
 nonempty = "0.10.0"
+prost = "0.12"
```

Comment:
probably better to make this an optional dependency, only enabled when the feature is enabled?